test/spec/browser_spec.rb

```diff
@@ -1,49 +1,19 @@
 require 'yaml'
-require_relative './app_spec.rb'
+require_relative './quicktest_spec.rb'
 
 config = YAML.load_file('config.yml')
-webdriver_url = "http://#{config['sauce_username']}:#{config['sauce_access_key']}@ondemand.saucelabs.com:80/wd/hub"
+browsers = YAML.load_file('browsers.yml')
 
-browser_definitions = {
-  safari_ios10_ipad: {
-    browserName: 'Safari',
-    deviceName: 'iPad Simulator',
-    platformName: 'iOS',
-    platformVersion: '10.0',
-    javascriptEnabled: true,
-    appiumVersion: '1.6.3'
-  },
-  chrome54_macos1012: {
-    browserName: 'chrome',
-    version: '54.0',
-    platform: 'macOS 10.12'
-  },
-  chrome54_windows10: {
-    browserName: 'chrome',
-    version: '54.0',
-    platform: 'Windows 10'
-  },
-  chrome48_linux: {
-    browserName: 'chrome',
-    version: '48.0',
-    platform: 'Linux'
-  }
-}
-
-def browsers_to_test
-  # TODO: put ENV vars in npm scripts; read here; return subset of browser_definitions.keys
-  [
-    :safari_ios10_ipad,
-    # :chrome54_macos1012,
-    # :chrome54_windows10,
-    # :chrome48_linux
-  ]
+browser_definitions = {}
+browsers.each do |browser|
+  browser_definitions[browser[0]] = browser[1]
 end
 
 describe "browser" do
-  browsers_to_test.each do |browser_id|
-     describe "#{browser_id}" do
-      include_examples "app", webdriver_url, browser_definitions[browser_id]
+  browser_definitions.each do |browser_id|
+    describe "#{browser_id}" do
+      browser_id[1]["browserstack.debug"] = true
+      include_examples "app", config['webdriver_url'], browser_id[1]
```

Comment:
I think it is better to have two keys since we can run 
`docker-compose -f docker-test.yml run web.test npm run test:quick`
`docker-compose -f docker-test.yml run web.test npm run test:integration`
`docker-compose -f docker-test.yml run web.test npm run test:browsers`
without changing the config file.

If we use the same key, every time we want to change from `:browsers` to `quick`or `integration` we need to change the config file.

But yes, we can use the same key. But we need to change the config time every time we need to change from `:browsers` to other test.